requirements.txt

```diff
@@ -17,7 +17,7 @@ paramiko==2.6.0
 pypiwin32==219; sys_platform == 'win32' and python_version < '3.6'
 pypiwin32==223; sys_platform == 'win32' and python_version >= '3.6'
 PySocks==1.6.7
-PyYAML==4.2b1
+PyYAML==5.2
```

Comment:
5.3 has been released, that would be nice to give it a try

Reply:
```suggestion
PyYAML==5.3
```

Reply:
👍